# For timestamps
Header header

# Desired positions
float64 lqATgt
float64 lqBTgt
float64 rqATgt
float64 rqBTgt
